app/main.py

```diff
@@ -1,8 +1,49 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self, comfort_class: int, clean_mark: int, brand: str):
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(self, distance_from_city_center: int, clean_power: int, average_rating: float, count_of_ratings: int):
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = round(average_rating, 1)
+        self.count_of_ratings = count_of_ratings
+
+    def calculate_washing_price(self, car: Car) -> float:
+        """Calculates the cost for a single car wash."""
+        if car.clean_mark < self.clean_power:
+            single_car_cost = (car.comfort_class
+                               * (self.clean_power - car.clean_mark)
+                               * self.average_rating
+                               / self.distance_from_city_center)
+            return round(single_car_cost, 1)
+        return 0.0
```

Comment:
Code Style: In the `calculate_washing_price` method, you have a complex mathematical operation that could be made more readable by splitting it into multiple steps or variables. This is a suggestion from the checklist. You can also consider adding comments to clarify the purpose of each step.